agency/README.md

```diff
@@ -84,7 +84,7 @@ _Path Parameters:_
 
 200 Success Response:
 
-If `device_id` is specified, `GET` will return an array with a single vehicle record, otherwise it will be a list of vehicle records with pagination details per the [JSON API](https://jsonapi.org/format/#fetching-pagination) spec:
+If `device_id` is specified, `POST` will return an array with a single vehicle record, otherwise it will be a list of vehicle records with pagination details per the [JSON API](https://jsonapi.org/format/#fetching-pagination) spec:
```

Comment:
I actually think these _are_ supposed to be `GET` endpoints. Lines `76` and `112` are incorrect.

Reply:
@marie-x could you take a look at this? There may be other get/post mix ups. I'm not fully sure how it's meant to work after the A/P reconciliation.

Reply:
@drtyh2o is correct, this is a copypasta error. Do you want me to do a full audit?

Reply:
Could you check the endpoints in just the Agency and Provider pages, and make sure the get/post is correct? You can do a commit to the branch in this PR.

Reply:
I've made the update in this commit https://github.com/openmobilityfoundation/mobility-data-specification/pull/837/commits/e7833680428d4ae4b4e99c8ff8fa38eb9b6a5f9b . Going to resolve this conversation, but feel free to comment if it's right or if it needs a change.